### PR TITLE
Fix checkstyle of GoLangFileMatch

### PR DESCRIPTION
**Pull request by tejksat**
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/653)

### CODE REVIEW

**Comment by tejksat** [comment]
@KostyaSha fix for the checkstyle fail.

**Comment by KostyaSha** [comment]
Problem was with method length, what are you trying to fix now?

**Comment by tejksat** [comment]
@KostyaSha it was one line more than valid. So I've transformed comments to exception messages. It seems to be ok now.

File: src/main/java/com/github/dockerjava/core/GoLangFileMatch.java [hunk 1]
```diff
@@ -154,8 +154,7 @@ private static String appendChunkPattern(StringBuilder patternStringBuilder, Str
                 case '[':
                     if (!isEsc) {
                         if (inRange) {
-                            //"[ is not expected, ] had not reached"
-                            throw new GoLangFileMatchException();
+                            throw new GoLangFileMatchException("[ not expected, closing bracket ] not yet reached");
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/GoLangFileMatch.java]: seem there is no test case for this case. Could you add?

**Pull request merged**